tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -152,3 +152,9 @@ Pull images from gcr.io
     Wait Until Keyword Succeeds  5x  15 seconds  Pull image  gcr.io/google_samples/gb-redisslave:v1
     Wait Until Keyword Succeeds  5x  15 seconds  Pull image  gcr.io/google_samples/cassandra:v11
     Wait Until Keyword Succeeds  5x  15 seconds  Pull image  gcr.io/google_samples/cassandra:v12
+
+Verify image manifest digest against vanilla docker
+    ${rc}  ${output}=  Run And Return Rc And Output  Pull image  busybox:1.26
```

Comment:
My bad - I don't think `Pull image` (in `Docker-Util.robot`) returns anything. We should probably just keep the original test logic.